workflows/prognostic_c48_run/runtime/transformers/tendency_prescriber.py

```diff
@@ -109,3 +104,33 @@ def step() -> Diagnostics:
 
         step.__name__ = func.__name__
         return step
+
+
+def scatter_within_tile(
```

Comment:
Is there some module where this function could live that both this module and `prescriber.py` import from?

Reply:
Yeah, we could make a new `scatter` module for this, don't think there's an existing one that makes sense. But probably a good idea in order to keep the import dag clean.